### ISSUE TITLE
SteamVR/ALVR not working when running through chroot

### ISSUE THREAD

**Issue by xenanthropy** [alvr-org/ALVR#1569]
## Description

Cannot get ALVR/SteamVR to work in a chroot. ALVR and steam both launch fine (steam works fine independently of ALVR) When launching SteamVR, ALVR does connect to my oculus quest 2 headset (it says 'connected' and displays IP and name of headset), but the alvr plugin gets disabled everytime, and I usually get an  "HMD not found error 108) each time. I've tested this on both an arch linux chroot and void linux chroot. For reference, I have a separate partition of void linux that I boot into and ALVR/SteamVR works fine in it.

## General Troubleshooting
- [X] I carefully followed the instructions in the [README](https://github.com/alvr-org/ALVR/blob/master/README.md) and successfully completed the setup wizard
- [X] I read the ALVR Wikis [here](https://github.com/polygraphene/ALVR/wiki) and [here](https://github.com/alvr-org/ALVR/wiki)

## Environment
Linux - running steam and ALVR through arch linux/void chroot, base machine is Sourcemage linux

### Hardware
**Note**: for Linux, an upload to the [`hw-probe`](https://linux-hardware.org/) database is preferred: `hw-probe -all -upload`
(the hw-probe is from outside of the chroot)
https://linux-hardware.org/?probe=1d232d1fed

**CPU**:
AMD Ryzen 7 5700X

**GPU**:
Radeon 5700 XT, amdgpu driver

**Audio**:
Pipewire

### Installation
**ALVR Version**:
ALVR v20.0.0-dev08+nightly.2023.04.07

**SteamVR Version**:
1.26.1 beta

**Install Type**:
- [ ] Packaged (`exe`, `deb`, `rpm`, etc)
- [X] Portable (`zip`) (The AppImage)
- [ ] Source

**OS Name and Version** (`winver` on Windows or `grep PRETTY_NAME /etc/os-release` on most Linux distributions):
Host machine: Source Mage GNU/Linux 0.63, kernel 6.1.6
chroot: Void Linux / Arch Linux

**Extra Details**
Before  anyone asks: the reason i'm needing to do this is because Sourcemage is a 64-bit only distribution, so steam cannot be natively installed. I normally just use flatpak to play steam, but of course ALVR doesn't work with flatpak steam, so I am trying to  use a chroot.
(Steam chroot was set up following [steam chroot gentoo guide](https://wiki.gentoo.org/wiki/Steam#Chroot), excluding gentoo-specific stuff, of course)

**Comment by mittorn** [comment]
try use ldd on driver_alvr_streamer library, maybe some dependency missing. SteamVR silently ignores driver if fails to load it

**Comment by mittorn** [comment]
I do not know where appimage mounted, but it should be under lib64/alvr/bin/linux64 folder relatively to alvr prefix
And it should be listed in external_drivers in ~/.config/openvr/openvrpaths.vrpath

**Comment by xenanthropy** [comment]
I decided to just grab the tarball, would be easier than figuring out the appimage: Well I checked both libalvr_vulkan_layer.so as well as driver_alvr_server.so. I was  missing libunwind for vulkan_layer so I grabbed that; For driver_alvr_server, i'm missing libx264.so.163 apparently... Vital part, i'm assuming lol. I will install that and report back

**Comment by xenanthropy** [comment]
Ah, I actually do have the codec, installed, but it's the wrong version... libx264.so.157. I'm assuming I need a newer one. Well I am on void linux chroot currently, i'll get arch linux setup again and check the libs on that.

**Comment by xenanthropy** [comment]
Ok, just to make sure Sourcemage wasn't causing the issue, I booted up into my void linux install (which 100% does work with alvr/steamvr) I set up the arch linux chroot and I got the exact same issues - alvr and gamepad plugin being auto-disabled, and no headset found error 108. So I can at least confidently say it is 100% a chroot issue and not something to do with my base system. 

I also wanted to test chrooting into my void linux install since I know alvr works in it, but sadly steam refuses to launch when I have it set up - keeps crashing.
